package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@remarkablemark for our code base when we turn on React Router 2.x we end up without any errors and no HTML returned in the res.render callback for the express view.  This was something we tried earlier before this changes was made with the same result.